tests/integration_tests.rs

```diff
@@ -27,41 +25,11 @@ mod unix {
 use unix::*;
 
 mod utils;
+use utils::command::{bat, bat_raw_command, bat_with_config};
```

Comment:
Can we fix the unused import warning here?

Reply:
I'm a bit confused. Where do you see a warning for this here? (I'm still pretty new to rust)

Reply:
I meant the warnings that are show in the PR diff view:

![image](https://user-images.githubusercontent.com/4209276/143768673-e6514e8e-05a2-4476-8802-82157abbca75.png)

Reply:
Oooh :smile: 
Yeah I didn't look there :smile: 
Thanks for pointing this out!